docs/use_case.md

```diff
@@ -0,0 +1,211 @@
+# Multiparty Computation for COVID-19 Training
```

Comment:
Use case should not just be a tutorial but talk of high level understanding of the possible real world use cases for prism, make references to the ai repo as well